code/src/steering_wheel_controller.cpp

```diff
@@ -0,0 +1,108 @@
+#include "../headers/steering_wheel_controller.hpp"
+
```

Comment:
Just a suggestion add namespace r2d2::manual_control saves some line length and makes it more readable imo

Reply:
Using is not allowed, the way to do it is to put all the functions within `namespace r2d2::manual_control{ CODE }` you don't have to do it per function all of the functions can be put inside it